packages/Adapters/Intrepid/src/DTK_IntrepidIntegrationRule.cpp

```diff
@@ -32,13 +32,13 @@
 */
 //---------------------------------------------------------------------------//
 /*!
- * \brief DTK_IntrepidEntityIntegrationRule.cpp
```

Comment:
Aww man I liked the "intrepIDENTITYintegrationwhatever". I meant to tease you about that.